content/en/real_user_monitoring/guide/tracking-rum-usage-with-usage-attribution-tags.md

```diff
@@ -46,21 +54,51 @@ RumMonitor.shared().addAttribute(forKey: "department", value: "marketing")
 
 **Note**: A few tags are included by default (`service`, `env`, `version`, `application.id`, and `application.name`). For anything else, set the global context using the method above.
 
-Once you've deployed this step, new RUM sessions are tracked according to the tags you added.
+After you've deployed this step, new RUM sessions are tracked according to the tags you added.
 
 ## View RUM usage
-The newly tagged sessions are displayed on the [Usage Attribution][3] page. When you review the RUM with Session Replay Sessions and RUM Sessions columns, you can see the number of sessions by department.
+The newly tagged sessions are displayed on the [Usage Attribution][1] page. When you review the RUM with Session Replay Sessions and RUM Sessions columns, you can see the number of sessions by department.
 
 {{< img src="real_user_monitoring/guide/rum-usage-attribution-tags/rum-use-attribution-tags-3.png" alt="View RUM usage by department" style="width:100%;">}}
 
-Usage information is also available through the [`GetHourlyUsageAttribution`][4] endpoint.
+Usage information is also available through the [`GetHourlyUsageAttribution`][5] endpoint.
+
+## Enforce tags to be set at the org level
+
+{{< callout url="#" btn_hidden="true" header="false">}}
+  Enforced RUM Usage Attribution is in Preview. To request access, contact <a href="https://docs.datadoghq.com/help/"> Datadog Support</a>.
+{{< /callout >}}
+
+You can require that a usage attribution tag gets added to a RUM application so that you can understand how its usage contributes to your Datadog bill. When this setting is active, usage attribution tags are required to be set upon creating a new or updating an existing RUM application in Datadog per organization.
```

Comment:
```suggestion
You can require that a usage attribution tag gets added to a RUM application so that you can understand how its usage contributes to your Datadog bill. When this setting is active, usage attribution tags are required to be set when creating a new RUM application or updating an existing RUM application in Datadog per organization.
```